package.json

```diff
@@ -24,14 +24,18 @@
     "grunt-contrib-jasmine": "2.0.1",
     "grunt-contrib-watch": "^1.0.0",
     "grunt-eslint": "20.1.0",
+    "grunt-exec": "3.0.0",
     "grunt-if-missing": "^1",
     "grunt-sass": "2.1.0",
+    "jasmine": "3.1.0",
     "jasmine-favicon-reporter": "1.0.0",
-    "load-grunt-tasks": "4.0.0"
+    "load-grunt-tasks": "4.0.0",
+    "puppeteer": "1.5.0"
```

Comment:
puppeteer is a dep of grunt-contrib-jasmine no?

Reply:
Yes, but I'm also explicitly using it for the end-to-end tests for extentRouter. So I think that it's best to declare that rather than assume that it's part of grunt-contrib-jasmine.